keyserver/src/main/scala/com/advancedtelematic/tuf/keyserver/data/KeyServerDataType.scala

```diff
@@ -30,11 +35,35 @@ object KeyServerDataType {
     require(keyType.crypto.validKeySize(keySize), s"Invalid keysize ($keySize) for $keyType")
   }
 
+  object SignedRootRole {
+    import io.circe.syntax._
+    import com.advancedtelematic.libtuf.data.ClientCodecs._
+
+    def fromSignedPayload(repoId: RepoId, payload: SignedPayload[RootRole]): SignedRootRole = {
+      val content = SignedPayload(payload.signatures, payload.signed, payload.json)
```

Comment:
Should the last parameter be `payload.signed.json`?

Reply:
`payload.signed` is a `RootRole` does not have a `json` attribute. `payload.json` is the json of the entire `payload`, so it would be the equivalent of `payload.asJson`

Reply:
Sorry I meant the equivalent of `payload.signed.asJson`

Reply:
Ah yeah I confused .json with .asJson.